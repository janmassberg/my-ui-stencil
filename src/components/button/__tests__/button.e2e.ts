import { newE2EPage } from "@stencil/core/testing";

describe("ui-button", () => {
  it("should set class names and aria-label attribute correctly", async () => {
    const page = await newE2EPage();
    await page.setContent(
      '<ui-button label="Label" kind="primary">Content</ui-button>',
    );
    const el = await page.find("ui-button.hydrated");
    expect(el).toMatchSnapshot();
  });
});
