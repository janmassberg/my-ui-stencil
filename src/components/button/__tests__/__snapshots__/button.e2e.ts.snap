// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ui-button should set class names and aria-label attribute correctly 1`] = `
<ui-button class="hydrated ui-button ui-button--primary ui-button--size-m" kind="primary" label="Label">
  <!---->
  <button aria-label="Label" class="ui-button__button">
    Content
  </button>
</ui-button>
`;
